tests/test_integration.py

```diff
@@ -144,7 +144,7 @@ def setUp(self):
             'int:basejessie:stable',
             'int:innogames:stable jessie',
         ]
-        obj['puppet_environment'] = None
+        obj['puppet_environment'] = 'make_serveradmin_chroot_unsafe'
```

Comment:
What is this?

Reply:
Just for now get get tests twice as fast. Will be gone, see that it is a separate commit.